Userland/Libraries/LibWeb/CSS/Parser/Parser.cpp

```diff
@@ -4725,6 +4726,85 @@ RefPtr<CSSStyleValue> Parser::parse_single_shadow_value(TokenStream<ComponentVal
     return ShadowStyleValue::create(color.release_nonnull(), offset_x.release_nonnull(), offset_y.release_nonnull(), blur_radius.release_nonnull(), spread_distance.release_nonnull(), placement.release_value());
 }
 
+RefPtr<CSSStyleValue> Parser::parse_rotate_value(TokenStream<ComponentValue>& tokens)
+{
+    // Value:	none | <angle> | [ x | y | z | <number>{3} ] && <angle>
+
+    // "none"
+    if (tokens.remaining_token_count() == 1) {
+        auto transaction = tokens.begin_transaction();
+        if (auto keyword = parse_keyword_value(tokens)) {
+            if (keyword->to_keyword() == Keyword::None) {
+                transaction.commit();
+                return keyword;
+            }
+        }
+    }
```

Comment:
We have a helper for this now: `parse_all_as_single_keyword_value(tokens, Keyword::None)`.